src/lib/utils/connection.go

```diff
@@ -9,7 +9,7 @@ import (
 func VerifyConnection() (bool, int) {
 	res, err := http.Get("http://clients3.google.com/generate_204")
 	if err != nil {
-		log.Fatalf("unable to ping external network\n")
+		log.Println("unable to ping external network")
```

Comment:
pls use  `log.Printf()` always instead of `log.Println()`. Please change the same in other lines as well.